### ISSUE TITLE
TypeError: i is undefined

### ISSUE THREAD

**Issue by ZeGuigui** [maxwroc/battery-state-card#68]
**Describe the bug**
In the logfile I have the following error:

```
2020-05-28 07:55:35 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
2020-05-28 07:55:40 ERROR (MainThread) [frontend.js.latest.202005194] https://my.homeassistant/hacsfiles/battery-state-card/battery-state-card.js:37:5224 TypeError: i is undefined
```

The card shows unknown battery level but battery_level attribute is set and has a value. One of the 6 entities is in unknown state but in the past only this one was shown as unknown.

**How to reproduce**
Add some zigbee devices to the card

**Expected behavior**
No error in the logfile and battery information showing in lovelace

**YAML configuration**
```yaml
entities:
  - attribute: battery_level
    entity: zigate.00158d0002897de3
    type: attribute
  - attribute: battery_level
    entity: zigate.00158d00028f323d
    type: attribute
  - attribute: battery_level
    entity: zigate.00158d0002c1472d
    type: attribute
  - attribute: battery_level
    entity: zigate.00158d0002c14c86
    type: attribute
  - attribute: battery_level
    entity: zigate.00158d0002c18425
    type: attribute
  - attribute: battery_level
    entity: zigate.00158d0002c8d9ae
    type: attribute
title: Batteries
type: 'custom:battery-state-card'
```

**Screenshots**
![Lovelace card](https://user-images.githubusercontent.com/13727111/83113674-43d29900-a0c8-11ea-9c2e-927e5f541025.png)

![Entity attributes](https://user-images.githubusercontent.com/13727111/83113744-58169600-a0c8-11ea-9d47-1569401e01c2.png)

**Version**
v1.3.5

**Comment by maxwroc** [comment]
Error is fixed in v1.3.6, thanks for reporting it. 

Though the problem in your case seems to be that the attribute is not found on entity. I see the listing which you have pasted but from the card point of view it seems that it was missing. The reason why it can be missing and why you could see these errors in logs is that the entity attributes are not there just after HA is launched/initialized. They are populated when the sensor sends the data for the first time (maybe that was your case).

The only explanation that comes to my mind now, to justify why the entities were not updating later on the card, is that the first entity was still not updated and since the error was happening it was stopping processing the rest.

Please let me know what is the state after update. If you still see the issue please look at the developer console in the browser (Ctrl+Shift+J / Command+Option+J) as it is easier to look there than in the logs.

![image](https://user-images.githubusercontent.com/8268674/83132890-f65f2780-a0d9-11ea-8f75-e5a0ffbdc3a6.png)

**Comment by ZeGuigui** [comment]
After update to 1.3.6 I get all the expected battery levels!
I just did a server restart to see if the levels return to unknown but this is not the case.
Thanks for the quick resolution of the bug!
